package-lock.json

```diff
@@ -3,6 +3,81 @@
   "requires": true,
   "lockfileVersion": 1,
```

Comment:
Something is off with this file too... Please revert and rerun "lerna bootstrap"

Reply:
Ok, sorry about that